packages/datadog-plugin-http/test/server.spec.js

```diff
@@ -131,5 +132,37 @@ describe('Plugin', () => {
         expect(() => res.emit('finish')).to.not.throw()
       })
     })
+
+    describe('with a blocklist configuration', () => {
+      beforeEach(() => {
+        return agent.load('http', { blocklist: '/health' })
+          .then(() => {
+            http = require('http')
+          })
+      })
+
+      beforeEach(done => {
+        const server = new http.Server(listener)
+        appListener = server
+          .listen(port, 'localhost', () => done())
+      })
+
+      it('should drop traces for blocklist route', done => {
+        const spy = sinon.spy(() => {
+          agent
+            .use((traces, spy) => {
```

Comment:
This should be calling the spy, otherwise this test will always pass even if there is a request to the agent.